microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+---
+title: Printing in WebView2 apps
+description: How to print a web page in a WebView2 app.
```

Comment:
```suggestion
description: How to print a web page to a printer, PDF file, or PDF stream in a WebView2 app.
```
opportunity to include search-engine terms and more broadly describe the capabilities.